go/examples/cmd/tutorial/main.go

```diff
@@ -85,29 +92,44 @@ func main() {
 
 	getTransactionsResponse, _, err := client.AnalyticsApi.AnalyticsGetTransactionsPost(ctx, getTransactionsRequest)
 	if err != nil {
-		util.HandleErrorAndExit(fmt.Errorf("couldn't get transactions: %v", err))
+		util.HandleErrorAndExit(fmt.Errorf("couldn't get transactions: %v", util.ErrorResponseString(err)))
 	}
 
 	fmt.Printf("Jane's transactions: %v\n", getTransactionsResponse)
+	newBankAddressRequest := sdk.GetNewAddressRequest{
+		WalletLabel: "bank",
+	}
+
+	newBankAddressResponse, _, err := client.WalletApi.WalletGetNewAddressPost(ctx, newBankAddressRequest)
+	if err != nil {
+		util.HandleErrorAndExit(fmt.Errorf("couldn't generate address for bank wallet: %v", util.ErrorResponseString(err)))
+	}
 
 	transferAssetRequest := sdk.TransferAssetRequest{
-		WalletLabel: "Jane",
-		RecipientAddress: "q1dxlf6vap2566t8w3z8f5j5lxy9n036zfsaytjve7fedsw6w8c9q9ctrwfz6ryyjwkgvj6tjg70f",
-		AssetId: 200,
-		Amount:  1,
-		Memo:    "Getting rid of my asset",
+		WalletLabel:      "Jane",
+		Authorization:    "123456",
+		RecipientAddress: newBankAddressResponse.RecipientAddress,
+		AssetId:          200,
+		Amount:           1,
+		Memo:             "Getting rid of my asset",
 	}
 
-	_, err = client.WalletApi.WalletTransferAssetPost(ctx, transferAssetRequest)
+	asyncTask, _, err = client.WalletApi.WalletTransferAssetPost(ctx, transferAssetRequest)
 	if err != nil {
-		fmt.Printf("Could not transfer asset. Does Jane have sufficient balance?\n")
+		util.HandleErrorAndExit(fmt.Errorf("couldn't transfer asset: %v", util.ErrorResponseString(err)))
 	}
 
-	time.Sleep(20)
+	transferTaskStatus, err := util.WaitForAsyncTaskDone(ctx, asyncTask.Id, client)
```

Comment:
What do you think about adding a new helper function with the following signature?
`util.WaitForAsyncTaskDone(ctx *Context, asyncTask *Task, client *Whatever)`

Then you won't need to think about the internals of `Task` when you use the function.

Reply:
There isn't `*Task` in the SDK.
There is `sdk.AsyncTaskCreatedResponse` or `sdk.GetTaskStatusResponse`.

Reply:
So I'm gonna leave it like that :)